Automattic/wp-calypso
# Guided onboarding: align intents with existing intent system and skip goals in site-setup

Fixes: 7381-gh-Automattic/dotcom-forge

~~Requires: D151653-code~~ Landed.

## Proposed Changes

This creates a new Stepper variant flow of `site-setup` called `site-setup-wg` that doesn't have the goals step, then redirects the users who went through Guided onboarding and declared their goals already towards it. 

## Why are these changes being made?
It doesn't make sense to re-ask people about their goals.

## Testing Instructions

1. Go to /start/guided.
3. Answer the survey and finish the flow.
4. You should land at `site-setup-wg` flow.
5. Finish the flow and take note of the site.
6. All should work.

#### Verify intent and goals are set
1. Go to https://YOUR_NEW_SITE/wp-admin/options.php
2. Verify the intent and goals are set.
